package.json

```diff
@@ -19,9 +19,12 @@
     "build": "react-scripts build",
     "eject": "react-scripts eject",
     "format": "prettier --write .",
+    "formatHusky": "prettier --write",
```

Comment:
Yes a test just with one, but when run, format all the files. It's because --write .
The dot cause the problem. In lint-staged object is selected with * all files but in the staged